cms/jinja2/templates/base.html

```diff
@@ -8,72 +8,80 @@
         {% if current_site and page.pk == current_site.root_page.pk and current_site.site_name %}{{ current_site.site_name }} - {% endif %}{% if page.seo_title %}{{ page.seo_title }}{% else %}{{ page.display_title | default(page.title) }}{% endif %}{% if current_site and page.pk != current_site.root_page.pk and current_site.site_name %} - {{ current_site.site_name }}{% endif %}
     {%- endwith -%}
 {%- endset -%}
 
 {% set navigation_settings = settings.navigation.NavigationSettings %}
-{% set main_menu = main_menu | default(navigation_settings.main_menu) %}
+{% set main_menu = main_menu | default(navigation_settings.main_menu.localized) %}
+
+{% set footer_menu = footer_menu | default(navigation_settings.footer_menu.localized) %}
 
-{% set footer_menu = footer_menu | default(navigation_settings.footer_menu) %}
+{#- We set variables for text so that it gets picked up by makemessages -#}
+{%- set badge_label = _("Beta") -%}
+{%- set badge_html = _("This is a new service.") -%}
+{%- set menu_label = _("Menu") -%}
+{%- set menu_label = _("Main menu") -%}
+{%- set menu_toggle_aria_label = _("Toggle main menu") -%}
+
+{%- set ogl_pre = _('All content is available under the') -%}
+{%- set ogl_text = 'Open Government Licence v3.0' -%}
+{%- set ogl_url = 'https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/' -%}
+{%- set ogl_post = _(', except where otherwise stated') -%}
 
 {% set pageConfig = {
     "bodyClasses": body_class,
     "title": page_title,
     "header": {
         "phase": {
-            "badge": 'Beta',
-            "html": 'This is a new service.'
+            "badge": badge_label,
+            "html": badge_html
         },
         "language": {
             "languages": languages
         },
         "mastheadLogoUrl": "/",
         "navLinks": {
             "id": "nav-links-external",
-            "ariaLabel": 'Main menu',
-            "ariaListLabel": 'Menu',
+            "ariaLabel": menu_label,
+            "ariaListLabel": menu_label,
             "toggleNavButton": {
-                "text": 'Menu',
-                "ariaLabel": 'Toggle main menu'
+                "text":menu_label,
+                "ariaLabel": menu_toggle_aria_label
             },
             "keyLinksList": main_menu_highlights(main_menu),
             "itemsList": main_menu_columns(main_menu)
         }
     },
     "footer": {
         "cols": footer_menu_columns(footer_menu),
-    "oglLink": {
-        "pre": _('All content is available under the'),
-        "text": 'Open Government Licence v3.0',
-        "url": 'https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/',
-        "post": _(', except where otherwise stated')
-    }
-    }
+        "oglLink": {
+            "pre": ogl_pre,
+            "text": ogl_text,
+            "url": ogl_url,
+            "post": ogl_post
+        }
+    },
+    "meta": {
+        "hrefLangs": hreflangs,
+    },
+    "page": page
```

Comment:
question: why is this needed here? If it is a remnant of previous iterations, can you please remove it? If not, add a clarifying inline comment